module-3/web-app/src/components/Pages/Documents/Documents.jsx

```diff
@@ -1,31 +1,57 @@
 import React, {useState, useEffect} from "react"
-import {useSearchParams} from "react-router-dom"
+import {useSearchParams, Link} from "react-router-dom"
 import style from "./Documents.module.scss"
 import mainStyles from "../../../App.module.scss"
 
-//TODO in progress documents catalogue page
+const DOCUMENT_STATUSES = {
+    DRAFT: {
+        title: "draft",
+        color: "#7f4fae"
+    },
+    IN_REVIEW: {
+        title: "in review",
+        color: "#b74e1b"
+    },
+    PUBLISHED: {
+        title: "published",
+        color: "#2b680f"
+    }
+}
+
 //TODO move this in component fetch to a redux action later on
 const Documents = () => {
 
     const [documents, setDocuments] = useState('')
     const [searchParams, setSearchParams] = useSearchParams()
     const projectId = searchParams.get('projectid')
-
+    
     useEffect(() => {
-        fetch("http://localhost:3001/selectAllDocuments/")
+        fetch(`http://localhost:3001/selectAllDocuments/${projectId}`)
```

Comment:
Use Redux to fetch documents, store documents in Redux state and use Redux selectors to query Redux state on documents page

Reply:
Done